### ISSUE TITLE
Problem with running rake tasks in test env

### ISSUE THREAD

**Issue by lucassus** [technicalpickles/rspec-spies#3]
```
ruby-1.9.2-p0$ RAILS_ENV=test bundle exec rake db:migrate:reset --trace
rake aborted!
undefined method `should' for module `RSpec::Matchers::BlockAliases'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rspec-expectations-2.6.0/lib/rspec/matchers/block_aliases.rb:4:in `alias_method'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rspec-expectations-2.6.0/lib/rspec/matchers/block_aliases.rb:4:in `<module:BlockAliases>'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rspec-expectations-2.6.0/lib/rspec/matchers/block_aliases.rb:3:in `<module:Matchers>'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rspec-expectations-2.6.0/lib/rspec/matchers/block_aliases.rb:2:in `<module:RSpec>'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rspec-expectations-2.6.0/lib/rspec/matchers/block_aliases.rb:1:in `<top (required)>'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:239:in `require'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:239:in `block in require'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:225:in `block in load_dependency'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:596:in `new_constants_in'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:225:in `load_dependency'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:239:in `require'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rspec-expectations-2.6.0/lib/rspec/matchers.rb:182:in `<top (required)>'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:239:in `require'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:239:in `block in require'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:225:in `block in load_dependency'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:596:in `new_constants_in'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:225:in `load_dependency'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/activesupport-3.0.9/lib/active_support/dependencies.rb:239:in `require'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rspec-spies-2.0.0/lib/rspec-spies.rb:18:in `<top (required)>'
/home/lucassus/.rvm/gems/ruby-1.9.2-p0/gems/bundler-1.0.15/lib/bundler/runtime.rb:68:in `require'
/home/lucassus/.rvm/gems/ruby-1.9.2-p0/gems/bundler-1.0.15/lib/bundler/runtime.rb:68:in `block (2 levels) in require'
/home/lucassus/.rvm/gems/ruby-1.9.2-p0/gems/bundler-1.0.15/lib/bundler/runtime.rb:66:in `each'
/home/lucassus/.rvm/gems/ruby-1.9.2-p0/gems/bundler-1.0.15/lib/bundler/runtime.rb:66:in `block in require'
/home/lucassus/.rvm/gems/ruby-1.9.2-p0/gems/bundler-1.0.15/lib/bundler/runtime.rb:55:in `each'
/home/lucassus/.rvm/gems/ruby-1.9.2-p0/gems/bundler-1.0.15/lib/bundler/runtime.rb:55:in `require'
/home/lucassus/.rvm/gems/ruby-1.9.2-p0/gems/bundler-1.0.15/lib/bundler.rb:120:in `require'
/home/lucassus/Projects/example-project/config/application.rb:7:in `<top (required)>'
/home/lucassus/Projects/example-project/Rakefile:4:in `require'
/home/lucassus/Projects/example-project/Rakefile:4:in `<top (required)>'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/rake_module.rb:25:in `load'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/rake_module.rb:25:in `load_rakefile'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/application.rb:495:in `raw_load_rakefile'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/application.rb:78:in `block in load_rakefile'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/application.rb:129:in `standard_exception_handling'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/application.rb:77:in `load_rakefile'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/application.rb:61:in `block in run'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/application.rb:129:in `standard_exception_handling'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/lib/rake/application.rb:59:in `run'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/gems/rake-0.9.2/bin/rake:32:in `<top (required)>'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/bin/rake:19:in `load'
/home/lucassus/Projects/example-project/vendor/gems/ruby/1.9.1/bin/rake:19:in `<main>'
```

**Comment by lucassus** [comment]
Also I have some problems with running specs with ci_reporter, see:

```
#bundle exec rake ci:setup:rspec spec CI_REPORTS=reports/ci_reporter
# TODO workaraung for ci_reporter and rspec-spy issue
bundle exec rspec spec --require=./vendor/gems/ruby/1.9.1/gems/ci_reporter-1.6.5//lib/ci/reporter/rake/rspec_loader.rb --format CI::Reporter::RSpec
```

**Comment by technicalpickles** [comment]
Looking at the stack trace, I'm only seeing one line of rspec-spies, which is just doing a require. I kind of suspect it may be an issue with rspec-expectations.

I'd suggest removing rspec-spies temporarily to see if that fixes the problem.

**Comment by lucassus** [comment]
Removing rspec-spies fixes this problem.

**Comment by lucassus** [comment]
this pull request https://github.com/technicalpickles/rspec-spies/pull/4 fixes this problem ;)

**Comment by jamesarosen** [comment]
I believe this should be fixed in 63caebd13e7f985a1c215895480a94657d036b0b. If it isn't, please reopen.
